internal/output/default.go

```diff
@@ -5,29 +5,24 @@
 package output
 
 import (
-	"errors"
 	"fmt"
 	"os"
-	"reflect"
+
+	"github.com/sirupsen/logrus"
 )
 
 // Default outputs the provided input to stdout.
 func Default(_input interface{}) error {
-	// check if the input provided is nil
-	if _input == nil {
-		return errors.New("empty value provided for default output")
-	}
+	logrus.Debugf("creating output with %s driver", DriverDefault)
```

Comment:
The log statements attempt to follow the same pattern for all the drivers for the sake of consistency